README.md

```diff
@@ -1,12 +1,10 @@
-aw-watcher-window
-=================
+# aw-watcher-window
 
 Cross-platform window-Watcher for Linux (X11), macOS, Windows.
 
 [![Build Status](https://travis-ci.org/ActivityWatch/aw-watcher-window.svg?branch=master)](https://travis-ci.org/ActivityWatch/aw-watcher-window)
 
 ### Note to macOS users
 
-To log current window title the teminal needs access to macOS accessibility API.
+To log current window title the terminal needs access to macOS accessibility API.
 This can be enabled in System Preferences > Security & Privacy > Accessibility, then add the Terminal to this list. If this is not enabled the watcher can only log current application, and not window title.
-
```

Comment:
Oops, the extra changes here were due to prettier, I believe. Let me know if you want them reverted. Not sure why it would remove the trailing line...